src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -612,20 +627,22 @@ public function partialRefundAction(int $orderId, Request $request)
      * @return RedirectResponse
      */
     #[AdminSecurity("is_granted('update', request.get('_legacy_controller')) && is_granted('delete', request.get('_legacy_controller'))")]
-    public function standardRefundAction(int $orderId, Request $request)
-    {
-        $formBuilder = $this->get('prestashop.core.form.identifiable_object.builder.cancel_product_form_builder');
-        $formHandler = $this->get('prestashop.core.form.identifiable_object.standard_refund_form_handler');
+    public function standardRefundAction(
+        int $orderId,
+        Request $request,
+        #[Autowire(service: 'prestashop.core.form.identifiable_object.builder.cancel_product_form_builder')] FormBuilderInterface $formBuilder,
+        #[Autowire(service: 'prestashop.core.form.identifiable_object.partial_refund_form_handler')] FormHandlerInterface $formHandler,
+    ) {
         $form = $formBuilder->getFormFor($orderId);
 
         try {
             $form->handleRequest($request);
             $result = $formHandler->handleFor($orderId, $form);
             if ($result->isSubmitted()) {
                 if ($result->isValid()) {
-                    $this->addFlash('success', $this->trans('A standard refund was successfully created.', 'Admin.Orderscustomers.Notification'));
+                    $this->addFlash('success', $this->trans('A standard refund was successfully created.', [], 'Admin.Orderscustomers.Notification'));
                 } else {
-                    $this->addFlashFormErrors($form);
+                    $this->addFlashErrors(explode("\n", (string) $form->getErrors(true)));
```

Comment:
```suggestion
                    $this->addFlashFormErrors($form);
```
Since you added the method in the parent class no need to change this, and at least it will maintain the same behaviour